ci/Jenkinsfile.android

```diff
@@ -2,7 +2,7 @@ pipeline {
   agent { 
     docker {
       label 'linux'
-      image 'statusteam/nix:jenkins-1.0.0-150d5721'
+      image 'statusteam/nix:jenkins-1.0.0-e20e5e15'
```

Comment:
these are a bit hard to track... can we maybe bump versions, like `1.0.1`, etc?

Reply:
Hopefully soon this won't be necessary. I want to make a docker image that will have part of its store as a volume on the host. So we start with fresh image very time, but don't have to re-download from cache every time.

Reply:
OK, we can start doing that from now on. For this PR it won't be beneficial since the image is already out there.